package.json

```diff
@@ -93,6 +93,7 @@
     "eslint-plugin-react": "^7.11.1",
     "eslint-plugin-unicorn": "^6.0.1",
     "firebase": "^5.7.3",
```

Comment:
Does `firebase-admin` supersede this, or do we still need plain `firebase`?  Asking because I noticed that you removed it from the documentation at storage.md.

Reply:
Not really.  `firebase` is the Web SDK and `firebase-admin` is the Admin SDK - they're independent (but sufficiently similar APIs).  I've only really retained the support for `firebase` in case there are existing users.  The tests exercise both, and the documentation describes both (although the example uses `firebase-admin` which I'm pretty sure is the better option).